src/static/css/_calls-to-action.css

```diff
@@ -0,0 +1,125 @@
+.calls-to-action--insight {
```

Comment:
Set the variables for the default here too:

```suggestion
.calls-to-action,
.calls-to-action--insight
```